styles.css

```diff
@@ -0,0 +1,183 @@
+a {
+  text-decoration: none;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  background-color: #303030;
+  display: flex;
+  justify-content: space-between;
+}
+
+.name {
+  color: white;
+  height: 24px;
+  font-family: Roboto, serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  position: relative;
+  top: 20px;
+  left: 10px;
+}
+
+.name p {
+  height: 24px;
+  width: 187px;
+}
+
+.burger {
+  height: 24px;
+  width: 20px;
+  margin-left: 40px;
+  margin-right: 5px;
+  margin-bottom: 15px;
+  border-radius: 0;
+  align-self: center;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+.burger a {
+  background-color: white;
+  margin-bottom: 5px;
+  padding-bottom: 0;
+}
+
+.main {
+  border-top: 1px solid slateblue;
+  color: white;
+  background-color: #151515;
+  width: 375px;
+  height: 610px;
+}
+
+.title {
+  margin-left: 20px;
+}
+
+.head {
+  margin-left: 20px;
+}
+
+.content {
+  margin-left: 20px;
+}
+
+h1 {
+  border-left: 20px;
+  color: #ff6b00;
+  height: 124px;
+  width: 327px;
+  left: 24px;
+  top: 120px;
+  font-family: Crete Round, serif;
+  font-size: 56px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  text-align: left;
+}
+
+.pretext {
+  height: 37px;
+  width: 327px;
+  left: 24px;
+  top: 256px;
+  font-family: Roboto, serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
```

Comment:
- [ ] Please use the required font-weight here which is `bold` instead of `700` in order to match your chosen design template.
- [ ] Also add the color `#FFF5E1` which is the required color as per your chosen template

Reply:
Hi, Julie,

Thank you for the detailed and clear comments, they were very helpful. However, regarding the style of this font - Roboto for "I’am a Software Developer", then Figma says 700 and not Bold. Here's a screenshot: 

![Screenshot from Figma](https://user-images.githubusercontent.com/52704856/143899043-6186368c-3459-442b-a86a-d7a05dec63c2.png).

Can I leave 700 or do you insist on bold?

Reply:
Hi @Gegardus,

I just confirmed that the font-weight is `700`. So sorry for any inconveniences this might have caused you 😄 
Please you can have the `700`.

Reply:
Thank you.

Reply:
You're welcome.